frontend/src/metabase/css/core/colors.css

```diff
@@ -5,20 +5,20 @@
   --color-brand: #509ee3;
   --color-brand-light: #ddecfa;
   --color-admin-navbar: #7172ad;
-  --color-white: #ffffff;
+  --color-white: #fff;
```

Comment:
This is debatable IMO. I prefer the 6-chars long color since it looks consistent with all other colors.